_security/configuration/disable-enable-security.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: default
+title: Disabling and enabling the Security plugin
+parent: Configuration
+nav_order: 40
+has_toc: true
+redirect_from: 
+ - /security-plugin/configuration/disable/
+---
+
+# Disabling and enabling the Security plugin
+
+The Security plugin is installed by default when using OpenSearch, but you can temporarily disable it or remove it altogether. Disabling the plugin involves a change to the `opensearch.yml` file, and you may want to do this to streamline testing. A more substantive change is to remove the Security plugin completely. You might want to remove it if, for example, you have your own security solution or need to remove it for development purposes. 
+
+Disabling or removing the plugin exposes the configuration index for the Security plugin. If the index contains sensitive information, make sure to protect it through some other means. If you no longer need the index, delete it.
+{: .warning }
+
+Disabling, removing, or installing the Security plugin requires a full cluster restart to enable security features.
+{: .warning}
+
+## Disabling/Enabling the Security plugin
+
+You can disable the Security plugin by editing the `opensearch.yml` file.   
+
+```yml
+plugins.security.disabled: true
+```
+You can then enable the plugin by removing the `plugins.security.disabled` setting.
+
```

Comment:
Is there anything they need to do on the Dashboard side when disabling the security plugin? See https://forum.opensearch.org/t/fails-on-ssl-when-using-rest-api-on-getting-started-local-docker-compose-cluster/8337.

Reply:
No. The only way to "disable" security on the Dashboard side is to remove both plugins.